packages/core/src/actors/observable.ts

```diff
@@ -35,7 +35,7 @@ export type ObservableActorLogic<
   TEmitted
 >;
 
-export type ObservableActorRef<TContext> = ActorRefFrom<
+type ObservableActorRef<TContext> = ActorRefFrom<
```

Comment:
Hmm, may want to make this one exported in `index.ts`

Reply:
![image](https://github.com/user-attachments/assets/e133333d-13e6-4b7e-89b7-7eb47cb3c522)

Caaaaaan dooooooo

I had actually gone the opposite direction and started adding all of them to `index.ts` but realized it'd be less effort to just remove them and add back the ones that are worth it.

Reply:
Resolved in #4976